lib/semian/circuit_breaker.rb

```diff
@@ -124,6 +124,7 @@ def log_state_transition(new_state)
       str << " success_count=#{@successes.value} error_count=#{@errors.size}"
       str << " success_count_threshold=#{@success_count_threshold} error_count_threshold=#{@error_count_threshold}"
       str << " error_timeout=#{@error_timeout} error_last_at=\"#{@errors.last}\""
+      str << " name=#{@name.inspect}"
```

Comment:
Could `@name` have spaces or other weird characters? Might be a good idea to take the approach of `error_last_at` and surround it with quotes.